devicemodel/hw/pci/virtio/virtio_block.c

```diff
@@ -544,7 +544,7 @@ virtio_blk_init(struct vmctx *ctx, struct pci_vdev *dev, char *opts)
 	pci_set_cfgdata16(dev, PCIR_VENDOR, VIRTIO_VENDOR);
 	pci_set_cfgdata8(dev, PCIR_CLASS, PCIC_STORAGE);
 	pci_set_cfgdata16(dev, PCIR_SUBDEV_0, VIRTIO_TYPE_BLOCK);
-	pci_set_cfgdata16(dev, PCIR_SUBVEND_0, VIRTIO_VENDOR);
```

Comment:
permanently change from REDHAT default VENDOR to ORACLE is not good,

Can we give an DM option in CMD line for this?

Reply:
Hi eddie

How about below CMD line for DM option, the ORACLE subsystem vendor id is used to launch WaaG with secure boot.

"       --virtio_sec_win: supports virtio-blk, virtio-net, virtio-input\n"
 "            and virtio-console for windows guest with secure boot\n",